typegen/strats/inline.py

```diff
@@ -90,7 +91,9 @@ def __init__(self, module: str, relevant: pd.DataFrame) -> None:
         self.df.loc[mask, Column.VARTYPE] = "None"
 
         self._module = module
-        self._scope_stack: list[cst.FunctionDef | cst.ClassDef] = []
+        self._scope_stack: list[cst.FunctionDef | cst.ClassDef | cst.Global] = []
+
```

Comment:
Globals don't have scopes. Fix this.

Reply:
Simply removed. Globals are tracked separately; this is a remnant of the first approach